### PR TITLE
DGJ_795-pdf-styling

### PR DESCRIPTION
**Pull request by iman-jamali-fw**
## Summary

This PR adds DGJ-specific pdf styling to `forms-flow-documents`. #795 

## Changes
- BC Sans font was added to the generated PDF
- DGJ-specific styles were added to the generated PDF
- A custom style was added to make a component hidden. A form designer can add `hidden-in-print` custom CSS to a component to make it hidden in the generated PDF.
- `forms-flow-documents` container support for reloading on file change was added for easier local development

## Screenshots (if applicable)

[Sample of an approved Telework PDF](https://github.com/bcgov/digital-journeys/files/10242212/Telework.agreement_sdf_Approved.pdf)

### CODE REVIEW

**Review by bhumin-fw — APPROVED** [review]
Looks good to me.
Just one question, if we make any CSS changes in the file src/styles.scss, will it reflect in PDF automatically?

**Comment by iman-jamali-fw** [comment]
@bhumin-fw unfortunately no, that change should be copied over to `forms-flow-document`.

File: forms-flow-documents/Dockerfile [hunk 1]
```diff
@@ -42,6 +43,10 @@ RUN : \
...
 RUN pip install -e .
 
+COPY ./lsyncd.conf lsyncd.conf
+
 EXPOSE 5006
-RUN chmod u+x ./entrypoint
-ENTRYPOINT ["/bin/sh", "entrypoint"]
+RUN chmod u+x ./entrypoint.dev
+
+CMD ["/bin/sh", "-c", "lsyncd lsyncd.conf && /bin/sh entrypoint.dev"]
```
Comment (simensma-fresh) [inline forms-flow-documents/Dockerfile:51]: Good call

**Review by simensma-fresh — APPROVED** [review]
Awesome!

**Comment by simensma-fresh** [comment]
Just out of curiosity, did you end up using the old css file or an updated one? Maybe we could add how to update / where that css file comes from to the documentation ticket

**Comment by iman-jamali-fw** [comment]
> Just out of curiosity, did you end up using the old css file or an updated one? Maybe we could add how to update / where that css file comes from to the documentation ticket

I didn't have a chance to try updating the CSS, I used the old one instead. Having the font working on the Flask template system was a time sink unfortunately :(

**Pull request merged**